src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java

```diff
@@ -0,0 +1,39 @@
+package harshadBJun20.assignment34;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/*Assignment - 34 :  25th July 2020
+ * Program 2 :
+Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4 */
+
+public class FindAndReplaceNameFromArrayList {
+	
+	void findAndReplaceName(ArrayList<String> namesList, String orgName, String replaceName) {
+		ArrayList<Integer> indexOfWord= new ArrayList<Integer>();
+		for(int index=0;index<namesList.size();index++) {
+				if(namesList.contains(orgName)) {
+					indexOfWord.add(namesList.indexOf(orgName));
+					namesList.set(namesList.indexOf(orgName), replaceName);
+				}				
+		}
+		System.out.println(orgName+" word index in given arrayList is "+indexOfWord);
+		System.out.println(namesList);
+		System.out.println("Total "+replaceName+" words :- "+(namesList.lastIndexOf(replaceName) - (namesList.indexOf(replaceName))));
```

Comment:
This is not correct logic. 
what if the below input is passed:
String[] names = {"Credits","Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"};

Reply:
ok Aastha Thanks. I will do necessary changes and update the PR.

Reply:
Changes Done and updated PR as per review comment. Please review. Thanks